pom.xml

```diff
@@ -156,22 +156,42 @@
               <goal>shade</goal>
             </goals>
             <configuration>
-             <createDependencyReducedPom>false</createDependencyReducedPom>
-             <artifactSet>
+              <createDependencyReducedPom>true</createDependencyReducedPom>
+              <promoteTransitiveDependencies>true</promoteTransitiveDependencies>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/services/org.jclouds.providers.ProviderMetadata</resource>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/services/org.jclouds.apis.ApiMetadata</resource>
+                </transformer>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                  <resource>META-INF/services/org.jclouds.logging.config.LoggingModule</resource>
+                </transformer>
+              </transformers>
+              <artifactSet>
                 <includes>
                   <include>org.eclipse.jetty:*</include>
+                  <include>com.google.guava:guava</include>
+                  <include>org.apache.jclouds:*</include>
```

Comment:
No need to relocate jclouds just needs to be shaded so it picks up the guava relocations